openmetadata-service/src/main/java/org/openmetadata/service/security/auth/LdapAuthenticator.java

```diff
@@ -268,7 +269,7 @@ private String getUserDnFromLdap(String email) {
             INTERNAL_SERVER_ERROR, MULTIPLE_EMAIL_ENTRIES, MULTIPLE_EMAIL_ENTRIES);
       } else {
         throw new CustomExceptionMessage(
-            INTERNAL_SERVER_ERROR, INVALID_USER_OR_PASSWORD, INVALID_EMAIL_PASSWORD);
+            INTERNAL_SERVER_ERROR, INVALID_USER_OR_PASSWORD, emailNotFound(email));
```

Comment:
hi @mgorsk1 , the reason we kept this way was not to allow email scrapping with brute, we don't even want to tell email not found

Reply:
this is as per our pentesting we do